src/llm/servable.cpp

```diff
@@ -38,6 +38,19 @@
 #include "text_utils.hpp"
 
 namespace ovms {
+
+void GenAiServable::determineDecodingMethod() {
+    getProperties()->decodingMethod = DecodingMethod::STANDARD;
+    auto& pluginConfig = getProperties()->pluginConfig;
+    if (pluginConfig.find("draft_model") != pluginConfig.end()) {
+        getProperties()->decodingMethod = DecodingMethod::SPECULATIVE_DECODING;
+    }
+    auto it = pluginConfig.find("prompt_lookup");
+    if (it != pluginConfig.end() && it->second.as<bool>() == true) {
+        getProperties()->decodingMethod = DecodingMethod::PROMPT_LOOKUP;
```

Comment:
what if there is both - draft_model and prompt_lookup? is it correct that in this case decoding method is overriden by PROMPT_LOOKUP?
Can one exist without other in both directions? prompt_lookup but without draft model? this is unhandled I think

Reply:
https://github.com/openvinotoolkit/openvino.genai/blob/master/src/cpp/src/continuous_batching/pipeline.cpp#L63